commands/help.js

```diff
@@ -28,16 +28,31 @@ module.exports = {
     let commandString = '```';
     let commandArray = [];
 
+/*    let authorRoles = [];
+    for (let role in message.author.roles){
+     authorRoles.push(role.get('name'));
+    }
+    console.log(authorRoles);*/
+
     // TODO: Display commands based on requireRoles
     for (let command in commands.commands) {
       let cmd = commands.commands[command];
       let info = '!' + command;
 
       // Skip commands that require roles for now
-      if (cmd.requireRoles) {
-        continue;
-      }
+      if (cmd.requireRoles){
+        let member = bot.guilds.first().members.get(message.author.id);
```

Comment:
My thought here was to search through the bot's guilds and look for any of them that a user has special perms on to determine if it should include privileged commands in help output. I know *our* bot only belongs to one guild, but that's be more **the right way™**.